File-Formats/PDF/Convert-HTML-To-PDF/mac.md

```diff
@@ -9,75 +9,73 @@ keywords: Assemblies
 
 # Convert HTML to PDF file in Mac
 
```

Comment:
Add description as like the aws.md file "The Syncfusion [HTML to PDF converter](https://www.syncfusion.com/pdf-framework/net/html-to-pdf) is a .NET library for converting webpages, SVG, MHTML, and HTML to PDF using C#. It is reliable and accurate. Using this library, you can convert HTML to PDF document in Mac."

Reply:
Added description